content/docs/how-tos/mirroring-to-quay.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Mirror an image to an external registry"
+date: 2020-10-05T11:14:39-04:00
+draft: false
+---
+
+
+## Requirements
+
+In order to mirror an image to quay, that image must be have a [promotion
+configuration.](/docs/architecture/ci-operator/#publishing-container-images)
+
+If the image is promoted into a namespace for which no other image mirroring is set up yet, some RBAC needs to
+be configured:
+
+* In [roles.yaml](https://github.com/openshift/release/blob/master/core-services/various_rbac/admin_ci-operator_roles.yaml),
+  add role `image-tagger` and rolebinding `image-tagger-ci-operator` for your namespace.
+* The admin of the namespace should allow the SA in the mirroring job defined below to access the images with `oc image mirror`,
+	e.g.,
+  [admin_openshift-kni_list.yaml](https://github.com/openshift/release/blob/master/core-services/image-mirroring/admin_openshift-kni_list.yaml)
+  which makes the images open to the public.
+
+## Configuring mirroring for new images
+
+
+Simply submit a PR adding the image to the appropriate mapping file. You will
+need an approval of an owner of the image set. Each folder in the [image mirroring
+config](https://github.com/openshift/release/tree/master/core-services/image-mirroring) is an image set.
+
+## Configuring mirroring for new sets of images
```

Comment:
```suggestion
## Configuring Mirroring for New Sets of Images
```

Reply:
Should probably add a check that all headings are in title case